finality-aleph/src/block/substrate/status_notifier.rs

```diff
@@ -64,6 +67,7 @@ impl ChainStatusNotifier<Header> for SubstrateChainStatusNotifier {
                 .map(|block| ChainStatusNotification::BlockImported(block.header))
                 .ok_or(Error::ImportStreamClosed)
             }
+            complete => Err(Error::AllStreamsTerminated)
```

Comment:
Is `complete` a keyword, or is something that I don't understand going on here?

Reply:
https://docs.rs/futures/latest/futures/macro.select.html